src/generators/concretes/genesisBlock.generator.ts

```diff
@@ -0,0 +1,48 @@
+import { Block, Transaction, TransactionType } from "@blockr/blockr-models";
+import { inject, injectable } from "inversify";
+import { BlockGeneratorException } from "../../exceptions";
+import { ConstantStore } from "../../stores";
+import { BlockGenerator } from "../abstractions/block.generator";
```

Comment:
Simplify import & alphabetically order import

```suggestion
import { BlockGenerator } from "..";
```